cloud_info/tests/utils.py

```diff
@@ -15,7 +15,9 @@ def get_variables_from_template(template, ignored_fields=[]):
     with open(template, "r") as f:
         content = f.read()
 
-    regexp = re.compile('%\((.+?)\)s')
+    # Look for variables names like
+    # ${static_compute_info['compute_service_production_level']}
+    regexp = re.compile('\${[^\[]+\[\'(.+?)\'\]}')
```

Comment:
I've been looking for a while to the regexp and decided to give up 😄 ...
I will assume it's good, maybe @brucellino wants to check also

Reply:
At least I was able to test that it works to match things expressed like in the example. :)
I found this website that works quite well with this regex: https://regex101.com/